### ISSUE TITLE
Segmentation for inset figures/tables

### ISSUE THREAD

**Issue by cambro** [UW-COSMOS/Cosmos#82]
This is pretty common in a few big-name publications. Example: http://birdnest.geology.wisc.edu/cosmos-tagger/view-training/037fcaac-14e5-43df-aba0-db61d3eceec9

Current segmentation step probably can't handle this.

**Comment by davenquinn** [comment]
Do we want these to be parsed out separately though? In at least this case, columns a) and b) are individually meaningful, but each graph is not...

**Comment by thodrek** [comment]
I would have it parsed separately but I would add an extra service to identify nested figures.

Theo

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by cambro** [comment]
Ok... seem to have a bug in the image tagger. that's not the right permalink!
@thodrek @davenquinn

**Comment by thodrek** [comment]
Regardless let's add the new subfigure enhancement request. Can you file a new one?
________________________________

**Comment by davenquinn** [comment]
Huh, how about that. Permalinks have always been tricky with the way the tagger is set up internally. Will create an issue to fix in visualizer.

**Comment by cambro** [comment]
Yeah, the permalinks actually seem to return a random image every time. Refresh a link and it keeps changing!

**Comment by davenquinn** [comment]
Yep, that is familiar from a while ago. I think we've fixed since, but our whole backing pipeline has changed dramatically since birdnest was deployed, and it'd be pretty time-costly to backport a fix. I'm prioritizing fixing permalinks for the newer instances Ian is spinning up.
